public/app/plugins/datasource/grafana/datasource.ts

```diff
@@ -60,7 +73,7 @@ class GrafanaDatasource extends DataSourceApi<any> {
       delete params.tags;
     } else {
       // require at least one tag
-      if (!_.isArray(options.annotation.tags) || options.annotation.tags.length === 0) {
+      if (!isArray(annotation.tags) || annotation.tags.length === 0) {
```

Comment:
nit: maybe use builtin Array.isArray function instead
```suggestion
      if (!Array.isArray(annotation.tags) || annotation.tags.length === 0) {
```